lib/webpacker/railtie.rb

```diff
@@ -24,7 +24,9 @@ class Webpacker::Engine < ::Rails::Engine
 
   initializer "webpacker.logger" do
     config.after_initialize do |app|
-      Webpacker.logger = ::Rails.logger
+      if ::Rails.logger.respond_to?(:tagged)
+        Webpacker.logger = ::Rails.logger
+      end
```

Comment:
Could create a tagged logger here to preserve the original logger:
```ruby
if ::Rails.logger.respond_to?(:tagged)
  Webpacker.logger = ::Rails.logger
else
  Webpacker.logger = ActiveSupport::TaggedLogging.new(::Rails.logger)
end
```

Reply:
I guess we are already doing this here right? https://github.com/rails/webpacker/blob/68b9f086c1d262be895c2750cb3a25879157733a/lib/webpacker/instance.rb#L2

Reply:
That creates a brand new logger. Doesn't wrap your existing `Rails.logger` instance.

Reply:
Ahh I see, totally missed 👍

Reply:
Makes sense... I'll push that today.